LayerChatTest/ViewControllers/Home/HomeViewController.swift

```diff
@@ -11,19 +11,24 @@ import UIKit
 import RxSwift
 import RxCocoa
 
-class HomeViewController : UIViewController {
+class HomeViewController: UIViewController {
     
     var viewModel: HomeViewModel
     let disposeBag = DisposeBag()
     
     @IBOutlet weak var conversationsTableView: UITableView!
     @IBOutlet weak var emptyLabel: UILabel!
     
+    var add: UIBarButtonItem
+    var logout: UIBarButtonItem
+    
     // MARK: Lifecycle methods
     
     init(viewModel: HomeViewModel) {
         self.viewModel = viewModel
-        
+        self.add = UIBarButtonItem(barButtonSystemItem: .add, target: nil, action: nil)
```

Comment:
why don't you use the buttons as optionals and create them after viewdDidLoad? I'd rather not do stuff like this in the init